apps/next/src/app/pages/single-page/single-page.ts

```diff
@@ -16,19 +16,19 @@
 
 import { Component } from '@angular/core';
 import { BaSinglePageContent } from '@dynatrace/shared/design-system/interfaces';
-import { BaPageService } from '@dynatrace/shared/data-access-strapi';
+import { DsPageService } from '@dynatrace/shared/data-access-strapi';
 
 @Component({
   selector: 'next-single-page',
   templateUrl: 'single-page.html',
   styleUrls: ['single-page.scss'],
   host: {
-    class: 'ba-page',
+    class: 'next-page',
   },
 })
 export class NextSinglePage {
```

Comment:
Remove the prefix on classes? This should be the default from the angular cli
@tomheller what do you think? 
```suggestion
export class SinglePage {
```